spec/factories/vm_azure.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :vm_azure, :class => "ManageIQ::Providers::Azure::CloudManager::Vm", :parent => :vm_cloud do
     location "westus"
-    vendor   "azure"
+    vendor   "microsoft"
```

Comment:
I believe that vendor needs to be "azure" because we already use "microsoft" for SCVMM.  And, I don't believe it's a good idea to overlap on vendor name.